lib/mix/tasks/gen.badges.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Mix.Tasks.Gen.Badges do
+  use Mix.Task
+  import Mix.Ecto
+
+  NimbleCSV.define(MyParser, separator: ";", escape: "\"")
```

Comment:
Most editors use ``,``instead of ``;`` any reason for this change?

Reply:
excel generated this with ; by default , we can pass de separator as a argument if you think its trully necessary